src/js/hca/config/form.js

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Should change this file to jsx or move the description element into helpers.

Reply:
@jbalboni great point, change made.